cmd/server/cmd/server.go

```diff
@@ -125,15 +122,21 @@ func runServer(sa *serverArgs, printf, fatalf shared.FormatFn) {
 	// get aspect registry with proper aspect --> api mappings
 	eval := expr.NewCEXLEvaluator()
 	adapterMgr := adapterManager.NewManager(adapter.Inventory(), aspect.Inventory(), eval, gp, adapterGP)
+	var store config.KVStore
+	store, err = config.NewStore(sa.configStoreURL)
+	if err != nil {
```

Comment:
Here and elsewhere you're forward-declaring the value so you don't shadow `err` with a `:=` assignment, presumably. Can we go ahead and collapse the call and the err check into the single `if err = foo; err != nil`?